src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, ToDraw {
+    private String color;
+
+    public Figure(Color color) {
+        this.color = String.valueOf(color);
```

Comment:
1. Keep the enum type as a field
2. Parse to string only if necessary - you can use `name()` method.